lntest/unittest/backend.go

```diff
@@ -59,6 +59,12 @@ func NewMiner(t *testing.T, netParams *chaincfg.Params, extraArgs []string,
 		t.Fatalf("unable to set up backend node: %v", err)
 	}
 
+	// Next mine enough blocks in order for segwit and the CSV package
+	// soft-fork to activate.
+	numBlocks := netParams.MinerConfirmationWindow*2 + 17
```

Comment:
Where did these numbers come from?

Reply:
partially from the itest where we mine blocks to activate it - then it's retry -> fail cycle to find the minimal blocks. Since it's not used anywhere else so I didn't make it into a constant.